types/edit-json-file/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for edit-json-file 1.4
+// Type definitions for edit-json-file 1.4.1
```

Comment:
Thank you for the update (though I haven't had anything to do with this library for a long time 😅), just this one thing is incorrect – looking at `edit-json-file` I see that the functions added here are 1.6.0+ features.
```suggestion
// Type definitions for edit-json-file 1.6.0
```

Reply:
Thank you!
I thougth i've to set it to the first version with this methods implemented.